### DISCUSSION TITLE
Does anyone know why this solution is marked as 100%?

### DISCUSSION THREAD

**Discussion by laxsjo** [cssbattle/forum#288] [Q&A]
I'm studying the current top solution of the open [target #120](https://cssbattle.dev/play/120),
which is as follows:
```html
<body bgcolor=0>▜▛      ▛▝<a>▝▝</a> ▟▙<p>█<style>html{color:54C144;border:solid+5vw}a{color:fff}*+*{font:5vw/25%a;margin:52%280-0%40}p{color:C74E4E;margin:-430%0;zoom:20;clip-path:pat\68('m0 0v4h10v2H0V2h4V0h2v2h4V0h2v9h2V0
```
When I use this in Chrome (version 104) the spaceship appears to be shifted a significant amount. The barrel also appears to be disconnected.

How it should appear:|How it appears:
:---:|:---:
![image](https://user-images.githubusercontent.com/72399173/184210306-ef3819fa-39c9-4f3d-ad43-b13963dbf1e7.png)|![image](https://user-images.githubusercontent.com/72399173/184210847-db729148-3c18-4a65-b50d-26e728c8fbd3.png)
**The diff:**|
![image](https://user-images.githubusercontent.com/72399173/184211006-62533833-c9c2-4d42-9f36-865add72ea39.png)|

Despite this, it's marked as 100% match. Now I know that the matching algorithm can be pretty lenient, but this feels like way too much. There's another explanation, right?

My first suspicion is that the version of chromium that is used on CSSBattles server is another version, leading to the rendered image being slightly different on the server. But this doesn't really feel like it explains it, considering that my version of chromium is the latest available version (as far as I can tell).

Does anyone have a good explanation for why this works?
Would greatly appreciate it!

**Comment by aarthvaen** [comment]
As CSSBattle mentions when you load a Unicode solution:

> HTML entities are rendered differently in different systems and might not work as you expect

That being said, CSSBattle uses your browser engine to render the solutions, which often can appear to not look perfect, yet they do the 100% match. But to know exactly how the server "sees" your solution, especially when working with Unicode, I recommend this previewer: https://tc70f3.csb.app

**Reply by laxsjo** [reply]
Thanks for the link
